### DISCUSSION TITLE
pollingInterval  question to control a no Smart Life outlet

### DISCUSSION THREAD

**Discussion by aketterson** [milo526/homebridge-tuya-web#178] [Q&A] [answered]
I have a Smart Life wall switch that I would like to use to control two Kasa outlets (connected to table lamps) though HomeKit automation.    The problem as I see it is that the it take too long for HomeKit to recognize that the Smart Life switch has changed state and trigger the automation to turn on/off the outlets?  Again, if I understand it correctly, the _pollingInterval_ variable sets the interval for HomeKit to poll the cloud and recognize a state change.  Per the documentation the minimum setting is for this interval 600 s — way too long.  Is there a better way to do this?

**Comment by milo526** [comment, marked as answer]
No.

Tuya limits how often and how fast this plugin can query your devices. This is set to once every 5 minutes.
Since I do not work for Tuya I cannot change this.

You might have better luck with the other homebridge plugins that support Tuya devices.
